pages/common/csvlook.md

```diff
@@ -0,0 +1,16 @@
+# csvlook
+
+> Render a CSV file in the console as a fixed-width table.
+> Included in csvkit.
+
+- View a CSV file:
+
+`csvlook {{data.csv}}`
+
+- View a CSV file with _less_ for easy scrolling:
```

Comment:
I realize this is a common example, but it doesn't really belong to this page as this one is more about `less` than `csvlook`.